design-system/materials/internals/definition.yaml

```diff
@@ -782,6 +804,14 @@ decisionGroupsByTheme:
           choice: '0 var(--spacing-20)'
         padding-for-tag:
           choice: '2px 12px'
+        padding-for-collapsible-panel-header:
+          choice: 'var(--spacing-30) 0'
```

Comment:
I am not so sure about this values of padding left and right = 0. It makes the side paddings for the header out of alignment.

<img width="613" alt="image" src="https://user-images.githubusercontent.com/19975842/205610960-f3e3776e-5243-4244-9657-f9c1bf9d97ed.png">

Reply:
Asking from Filip, this is actually the expected behaviour.